templates/driver/README.md.tmpl

```diff
@@ -0,0 +1,25 @@
+# { name | upper } Driver [![crates.io](https://img.shields.io/crates/v/{ name }.svg)](https://crates.io/crates/{ name }) [![Documentation](https://docs.rs/{ name }/badge.svg)](https://docs.rs/{ name }) ![CI Build](https://github.com/braun-embedded/step-dir/workflows/CI%20Build/badge.svg)
+
+## About
+
+Rust driver crate for the [{ name | upper }] stepper motor driver. Carrier boards for this chip are [available from Pololu].
+
+This crate is a specialized facade of the [Step/Dir] library. Please consider using Step/Dir directly, as it provides drivers for more stepper motor drivers, as well as an interface to abstract over them.
```

Comment:
Not really on topic here (I suppose that's the same text as before), but this sounds better to me:
```suggestion
This crate is a specialized facade for the [Step/Dir] library. Please consider using Step/Dir directly, as it provides drivers for more stepper motor drivers, as well as an interface to abstract over them.
```

Feel free to disregard, I'm not the native speaker here :-)